src/libslic3r/Support/TreeSupport.cpp

```diff
@@ -141,9 +141,6 @@ static std::vector<std::pair<TreeSupportSettings, std::vector<size_t>>> group_me
 #ifndef NDEBUG
         const PrintObjectConfig &object_config = print_object.config();
```

Comment:
Will this still be used if those asserts are removed?

Reply:
Don’t think this is related - only if the debug